app/scripts/lib/constants.js

```diff
@@ -15,9 +15,10 @@ define([], function () {
     FX_DESKTOP_CONTEXT: 'fx_desktop_v1',
     FX_DESKTOP_SYNC: 'sync',
 
+    VERIFICATION_POLL_IN_MS: 4000,
     SIGNUP_RESEND_MAX_TRIES: 3,
     PASSWORD_RESET_RESEND_MAX_TRIES: 3,
-    RESET_PASSWORD_POLL_INTERVAL: 2000,
```

Comment:
To be clear, is this intentional cleanup of an unused value?  IMO these are better done as separate commits, but no big deal.